src/common/types/survey.type.ts

```diff
@@ -1,5 +1,9 @@
 import { SurveyStatus } from './survey-status.type';
 
+export function surveyDateToLocaleString(unixtime: number): string {
```

Comment:
This is applicable to any date, not just `surveyDate` like the function name implies

I think it's better to put this in a `common/utils/date.utils.ts` file since it doesn't belong in a `.type` file

Reply:
Updated in b7528671d16ce8e61b8e7cfa7723f034c8441117